haas/rest.py

```diff
@@ -123,6 +125,10 @@ def rest_call(method, path, schema):
             argument to appear in both the request body and the path; such
             requests will be rejected prior to invoking the schema.
 
+            Because GET query parameters have no type, ensure that any GET
+            arguments of non-string type implement Schema's "Use" to perform
+            type validation and conversion.
```

Comment:
It took a little digging to figure out the "Use" function.

Take a look at the new verbiage and let me know what you think.